barretenberg/cpp/src/barretenberg/proof_system/arithmetization/arithmetization.hpp

```diff
@@ -151,16 +151,26 @@ template <typename FF_> class UltraArith {
     struct TraceBlocks {
         UltraTraceBlock pub_inputs;
         UltraTraceBlock arithmetic;
-        UltraTraceBlock sort;
+        UltraTraceBlock genperm;
```

Comment:
"sort" is too loaded

Reply:
I agree, but now it doesn't show the logic of the relation (which is ordering the values). Maybe we should pick another name? Genperm makes me think that it is somehow related to a permutation, but the relation itself has nothing to do with permutations :(

Reply:
Changed this here to delta_range and made a TODO to update the name from GenPermSort to DeltaRangeConstraint everywhere else (since the diff becomes pretty large on that)